test/pkg/ptphelper/ptphelper.go

```
@@ -1261,43 +1209,262 @@ func IsPTPOperatorVersionAtLeast(minVersion string) bool {
+	}
+	port := os.Getenv("GNSS_SIM_API_PORT")
+	if port == "" {
+		port = "9200"
+	}
+	return fmt.Sprintf("http://%s:%s", host, port)
+}
+
+// GNSSSimSignalLoss calls POST /api/signal/loss on the gnss-sim, equivalent
+// to ubxtool CFG-NAVSPG-INFIL_NCNOTHRS,50,1 on real hardware.
+func GNSSSimSignalLoss() error {
+	resp, err := http.Post(GNSSSimAPIBase()+"/api/signal/loss", "application/json", nil)
```

golangci-lint (2.12.2)

[error] net/http.Post must not be called. use net/http.NewRequestWithContext and (*net/http.Client).Do(*http.Request)

(noctx)